test/utils.test.js

```diff
@@ -492,7 +496,7 @@ describe('utils', function() {
         b: 'asd',
         c: true,
         d: undefined,
-        e: 'very long string that is definitely ove\u2026',
+        e: 'very long string that is definitely over\u2026',
```

Comment:
do you care about this off by one?

Reply:
No, I didn't realize implementation of `truncate` that we already had didnt count ellipsis as the character so I just unified it.